src/cltk/embeddings/processes.py

```diff
@@ -38,7 +47,7 @@ class EmbeddingsProcess(Process):
     language: str = None
     variant: str = "fasttext"
     embedding_length: int = None
-    word_idf: Optional[Dict[str, float]] = field(repr=False, default=None)
+    idf_model: Optional[Dict[str, float]] = field(repr=False, default=None)
```

Comment:
I don't think the optional markers are need on these three items.

Reply:
My understanding: If a value can ever be `None`, then it needs to be optional. The default param value for all three is `None`, hence the Optional. I'd like to keep this but would revisit anytime.